pkg/sampling/strategystore/strategy.go

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2018 The Jaeger Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package strategystore
+
+// Strategy defines a sampling strategy. Type can be "probabilistic" or "ratelimiting"
+// and Param will represent "sampling probability" and "max traces per second" respectively.
+type Strategy struct {
```

Comment:
these types look like private intermediate model used by static impl. Or did you mean the StrategyStorage interface to return these?

Reply:
made private